README.rst

```diff
@@ -104,11 +104,41 @@ on the command line:
 | | ``--json``      | Deep tokenization. Output token objects in JSON   |
 |                   | format, one per line.                             |
 +-------------------+---------------------------------------------------+
-| | ``--normalize`` | Normalize punctuation, causing e.g. quotes to be  |
+
+Other options can be specified on the command line:
+
++-------------------+---------------------------------------------------+
+| | ``--n``         | Normalize punctuation, causing e.g. quotes to be  |
```

Comment:
I think the correct form, by convention, is `-n`, not `--n`. Short-form arguments are specified with a single hyphen, while long-form ones use double hyphens.

Reply:
Done!